consul_awx.py

```diff
@@ -83,10 +85,17 @@ def build_full_inventory(self, node_meta=None, tagged_address="lan"):
                 if not value:
                     continue
 
-                try:
-                    value = str2bool(value.strip())
-                except ValueError:
-                    pass
+                # Keep from converting some values to boolean
+                # So a valid value of 1 or 0 is kept as is
+                convert_to_bool = True
+                if node_meta_types and key in node_meta_types:
```

Comment:
`convert_to_bool` is used only one time below.
I reckon we can change a little bit the if statement to not need it.
Something like
```python
if not (node_meta_types and key in node_meta_types):
    try:
        value = str2bool(value.strip())
    except ValueError:
        pass
```